packages/parser/lib/util.re

```diff
@@ -0,0 +1,9 @@
+module Location = Ppxlib.Location;
+
+let make_loc =
+    (~loc_ghost=false, start_pos: Lexing.position, end_pos: Lexing.position)
```

Comment:
Do you think this file can be called Parser_location and have a make function instead?

Reply:
Yes, that is fine assuming that it will only serve that purpose. Otherwise, I named it so, in case we ever need any other helper function that won't be constrained to one use purpose.

Reply:
Right, I prefer to not have non-ambigious names such as Utils on this codebase, and keep the module focused on one thing

Reply:
Done in https://github.com/davesnx/styled-ppx/pull/369/commits/70f8948835d278236b3662361951f2a353c9f2d8